central/networkgraph/entity/networktree/manager_bench_test.go

```diff
@@ -31,25 +33,40 @@ func BenchmarkCreateNetworkTree(b *testing.B) {
 		entities[i] = e
 		ip = nextIP(ip)
 	}
+	entitiesByCluster := map[string][]*storage.NetworkEntityInfo{c1: entities}
 
 	// Above data will create one of the worst possible tree since each CIDR is disjoint aka all nodes are leaves,
 	// hence resulting in comparison with every node for each new entry.
 
-	mgr := newManager()
 	b.Run("createNetworkTree", func(b *testing.B) {
-		err := mgr.Initialize(map[string][]*storage.NetworkEntityInfo{"c1": entities})
-		require.NoError(b, err)
+		for i := 0; i < b.N; i++ {
+			t := createNetworkTree(b, entitiesByCluster)
```

Comment:
This is benchmarking more than initial network tree creation. The call to create network tree for c2 is not relevant to this scenario.

Reply:
Oh, I thought this is what we want to check based on the benchmark name. Do you think we should split it to separetly test 
- initialize
- createNewtorkTree
- Insert

Reply:
I see how it can be interpreted differently. Yes, it would be good to keep the tests scoped to at least init (create from dataset) and upserts.